### PR TITLE
Add requirement and basic test for Prescient

### PR DESCRIPTION
**Pull request by lbianchi-lbl**
## Summary/Motivation:

- Prescient is required for the majority (if not all) of upcoming milestones/deliverables

## Changes proposed in this PR:

- Add Prescient (`gridx-prescient`) as a requirement in `setup.py`
- Add a basic test (with data files) that can be used to detect whether Prescient has been set up correctly independently from the client code that will be developed in DISPATCHES

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.md and COPYRIGHT.md file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

**Review by bknueven — APPROVED** [review]
This looks good.

We should maybe set up a Prescient regression baseline in DISPATCHES, just so as we're developing tests for our workflows developers have an example they can refer to.

**Comment by lbianchi-lbl** [comment]
- [ ] Add a README with a link to the data files in the Prescient repo

File: dispatches/tests/data/prescient_5bus/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Notice about these data files
+
+These data files originate from Prescient's [GitHub repository](https://github.com/grid-parity-exchange/Prescient/tree/main/examples/5bus), and are subject to Prescient's [license and copyright](https://github.com/grid-parity-exchange/Prescient/blob/main/LICENSE.txt).
+Their use within DISPATCHES is limited to testing its Prescient requirements in isolation from any other DISPATCHES-specific functionality.
```
[the remark this answers is not in the document]
Comment (lbianchi-lbl) [reply dispatches/tests/data/prescient_5bus/README.md:4]: Thanks @jsiirola, this looks much better.

**Comment by ksbeattie** [comment]
@adowling2 review?

**Review by bknueven — APPROVED** [review]

**Review by adowling2 — APPROVED** [review]

**Pull request merged**